frontend/src/assets/i18n/translations/en.json

```diff
@@ -43,17 +43,18 @@
         "edit": "Edit",
         "close": "Close Survey",
         "edit_warning": "Do you really want to edit this survey? All participants have to answer them again!",
-        "edited": "The survey \"{{ title }}\" was edited. You have answer the again",
-        "timeslotSelected": "The survey is fixed on {{start}} - {{end}}",
-        "fix_appointment": "Do you want to select this date?",
+        "edited":
+        "The survey \"{{ title }}\" was edited. You need to answer it again to make sure everything is alright.",
+        "timeslotSelected": "The survey chose the following date: {{start}} - {{end}}",
```

Comment:
I just noted, that this won't work with timeslots without and end. I added that to the Frontend issue document.